### ISSUE TITLE
Pip install problem

### ISSUE THREAD

**Issue by chrisgilmerproj** [muricoca/crab#78]
Currently I get this when trying to install crab:

```
error in scikits.crab setup command: Distribution contains no modules or packages for namespace package 'scikits'
```

My installation attempts included

```
pip install git+git://github.com/muricoca/crab.git
```

and 

```
pip install -e git+git://github.com/muricoca/crab.git#egg=crab
```

**Comment by chrisgilmerproj** [comment]
See fix in #81.

**Comment by qige96** [comment]
I suggest that you should give up this package because it has not been maintained for years. If you need a python recommender engine, you can try python-surprise.
